components/pricing_modal/downgrade_team_removal_modal/index.tsx

```diff
@@ -0,0 +1,220 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useEffect, useState} from 'react';
+import {FormattedMessage, useIntl} from 'react-intl';
+import {isEmpty} from 'lodash';
+import {Modal} from 'react-bootstrap';
+
+import {useDispatch, useSelector} from 'react-redux';
+
+import {t} from 'utils/i18n';
+
+import RadioButtonGroup from 'components/common/radio_group';
+import DropdownInput, {ValueType} from 'components/dropdown_input';
+
+import useGetUsage from 'components/common/hooks/useGetUsage';
+import {getTeams, selectTeam, archiveAllTeamsExcept} from 'mattermost-redux/actions/teams';
+import SuccessModal from 'components/success_modal';
+import {getActiveTeamsList} from 'mattermost-redux/selectors/entities/teams';
+import {closeModal, openModal} from 'actions/views/modals';
+import {ModalIdentifiers} from 'utils/constants';
+import {isModalOpen} from 'selectors/views/modals';
+import {GlobalState} from 'types/store';
+import {subscribeCloudSubscription} from 'actions/cloud';
+import {fallbackStarterLimits, asGBString} from 'utils/limits';
+
+import './downgrade_team_removal_modal.scss';
+
+type Props = {
+    onHide?: () => void;
```

Comment:
When opening this, only product_id and starterProductName are passed as dialogProps.

Reply:
Removed